packages/scene-composer/src/utils/entityModelUtils/SceneLifecycleModule.ts

```diff
@@ -0,0 +1,40 @@
+import { GetSceneCommandOutput } from '@aws-sdk/client-iottwinmaker';
+
+import {
+  LAYER_ROOT_ENTITY_ID,
+  LAYER_ROOT_ENTITY_NAME,
+  SCENE_ROOT_ENTITY_COMPONENT_NAME,
+  DYNAMIC_SCENE_CAPABILITY,
+} from '../../common/entityModelConstants';
+import { getGlobalSettings } from '../../common/GlobalSettings';
+import { KnownSceneProperty } from '../../interfaces';
+import { emptyScene } from '../emptyScene';
+
+import { updateSceneEntityComponent } from './sceneComponent';
+
+export class SceneLifecycleModule {
+  createDynamicScene = async (): Promise<GetSceneCommandOutput | Error> => {
```

Comment:
it should have `SceneMetadataModule` as a parameter, and return type should just be `Promise<GetSceneCommandOutput>`. any error should be throw instead of return in the promise

Reply:
Added and pushed!